### PR TITLE
Added traceability details

### PR DESCRIPTION
**Pull request by sha31dev**

### CODE REVIEW

File: lipton_custom/fixtures/custom_field.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "dt": "Batch",
```
Comment (SaiFi0102) [inline lipton_custom/fixtures/custom_field.json:14]: Need to export fixtures again. traceability_details not included in this file

File: lipton_custom/hooks.py [hunk 2]
```diff
@@ -9,16 +9,19 @@
 
 
 fixtures = [
-	{
-		"doctype": "Custom Field",
-		"filters": {
-            "name": [
+    {
```
Comment (SaiFi0102) [inline lipton_custom/hooks.py:12]: these are all spaces
image

File: lipton_custom/fixtures/custom_field.json [hunk 3]
```diff
@@ -78,19 +78,19 @@
...
   "in_list_view": 0,
   "in_preview": 0,
-  "in_standard_filter": 1,
-  "insert_after": "garden_mark",
+  "in_standard_filter": 0,
+  "insert_after": "grade",
   "is_system_generated": 0,
   "is_virtual": 0,
-  "label": "Grade",
+  "label": "Column Break",
```
Comment (SaiFi0102) [inline lipton_custom/fixtures/custom_field.json:85]: Gotta keep the labels empty for Column Break fieldtypes

File: lipton_custom/fixtures/custom_field.json [hunk 4]
```diff
@@ -198,6 +198,59 @@
...
+  "ignore_user_permissions": 0,
+  "ignore_xss_filter": 0,
+  "in_global_search": 0,
+  "in_list_view": 0,
+  "in_preview": 0,
+  "in_standard_filter": 0,
+  "insert_after": "taste",
+  "is_system_generated": 0,
+  "is_virtual": 0,
+  "label": "Column Break",
```
Comment (SaiFi0102) [inline lipton_custom/fixtures/custom_field.json:244]: Gotta keep the labels empty for Column Break fieldtypes

File: lipton_custom/fixtures/custom_field.json (hunk 1 above)
Comment (sha31dev) [reply lipton_custom/fixtures/custom_field.json:14]: Left out due to a type. There was a missing ','.

File: lipton_custom/fixtures/custom_field.json (hunk 3 above)
Comment (sha31dev) [reply lipton_custom/fixtures/custom_field.json:85]: Updated in new PR

**Pull request merged**